dev-python/vector/vector-1.6.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+#
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+DISTUTILS_USE_PEP517=hatchling
+inherit distutils-r1 pypi optfeature
+
+DESCRIPTION="Vector classes and utilities"
+HOMEPAGE="
+	https://github.com/scikit-hep/vector
+	https://vector.readthedocs.io/
+	https://doi.org/10.5281/zenodo.7054478
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+# tests need numba
+RESTRICT="test"
+
+RDEPEND="
+	>=dev-python/numpy-1.13.3[${PYTHON_USEDEP}]
+	>=dev-python/packaging-19[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	dev-python/hatch-vcs[${PYTHON_USEDEP}]
+"
+
+export SETUPTOOLS_SCM_PRETEND_VERSION=${PV}
```

Comment:
Let's put this in a consistent place, personally I like having it at the top with the distutils/python variables. I also think the `export` is not required here (but I see it often, so not completely sure what is going on with this one).